mk/xe-linux-distribution

```diff
@@ -478,7 +478,6 @@ identify_yinhe()
     fi
     
     yinhe_osname="Yinhe Kylin Linux ${major}.${minor}"
-    write_to_output "${distro}" "${major}" "${minor}" "${yinhe_osname}"
```

Comment:
why this removal?

Reply:
Genuine goof. I will commit a version that puts that back.